### PR TITLE
SASS-7878: Audit events for property rentals income AND expenses

### PR DESCRIPTION
**Pull request by vivekhmrc**

### CODE REVIEW

File: app/audit/PropertyRentalsExpense.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * limitations under the License.
+ */
+
+package audit
+
+import pages.PageConstants
+import play.api.libs.json.{JsPath, Json, OFormat}
+import queries.Gettable
+
+case class PropertyRentalsExpense(consolidatedExpensesYesNo: Option[Boolean],
```
Comment (arunkumar461) [inline app/audit/PropertyRentalsExpense.scala:23]: consolidatedExpenses is an object; you may need to include the amount as well

File: app/audit/PropertyRentalsIncome.scala [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * limitations under the License.
+ */
+
+package audit
+
+import pages.PageConstants
+import play.api.libs.json.{JsPath, Json, OFormat}
+import queries.Gettable
+
+case class PropertyRentalsIncome(calculatedFigureYourself: Option[Boolean],
```
Comment (arunkumar461) [inline app/audit/PropertyRentalsIncome.scala:23]: calculatedFigureYourself is an object, same here

**Review by arunkumar461 — CHANGES_REQUESTED** [review]
just enable debug logs and share the JSON within the group

**Review by modivilioglu — APPROVED** [review]
LGTM

**Pull request merged**